### DISCUSSION TITLE
How do you manage constant STORE_DIR growth of @pnpm/package-store?

### DISCUSSION THREAD

**Discussion by OverLord561** [pnpm/pnpm#8319] [General]
Hi Folks!

We use  "@pnpm/package-store": "^19.0.11", to store packages on server-side to optimize Server response time.

The problem is that the number of packages is huge, leading to STORE_DIR growth. How do you guys keep STORE_DIR healthy and small? Is there some kind of strategy, like taking the server off the load balancer, cleaning the folder, and then putting the server back on the load balancer?

Is there some kind of eviction policy?

Thanks in advance

**Comment by OverLord561** [comment]
Hi @zkochan , do you have any thoughts on this? I'd really appreciate your input.

**Reply by zkochan** [reply]
We don't have such use case. We don't maintain a registry.

**Reply by OverLord561** [reply]
Thank you very much for the input!
